### PR TITLE
Introduction of Id in Learning Content for Viva Learning Workload

### PR DESCRIPTION
**Pull request by SritejaKampara**

### CODE REVIEW

File: api-reference/beta/api/learningcontent-get.md [hunk 1]
```diff
@@ -33,6 +33,10 @@ One of the following permissions is required to call this API. To learn more, in
 GET /employeeExperience/learningProviders/{learningProviderId}/learningContents/{externalId}
 ```
 
+``` http
+GET /employeeExperience/learningProviders/{learningProviderId}/learningContents/{learningContentId}
+```
+
```
Comment (isvargasmsft) [inline api-reference/beta/api/learningcontent-get.md:39]: General:
In PR https://github.com/microsoftgraph/microsoft-graph-docs/pull/17789 we are modifying the HTTP request format. Could you please explain this additional HTTP format? Are these changes articulated?

File: api-reference/beta/api/learningcontent-update.md [hunk 2]
```diff
@@ -146,6 +153,7 @@ Content-Type: application/json
 
 {
     "@odata.context": "https://graph.microsoft.com/beta/$metadata#learningProviders('13727311-e7bb-470d-8b20-6a23d9030d70')/learningContents/$entity",
+    "id": "77029588-a660-46b6-ba58-3ce4d21d5678",
```
Comment (isvargasmsft) [inline api-reference/beta/api/learningcontent-update.md:156]: Was there a schema change regarding the id properties of the learningContent resource? If so, would you be so kind to share the CSDL PR on the side, please?

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Hello @SritejaKampara, 

Thank you very much for your work in this PR. 

Just a few questions and suggestions:
1. Please clarify if this additional HTTP request format is articulated in any way with the change introduced in PR 17789. Does this operation has 2 possible HTTP request formats?
2. Please clarify if there was an schema change regarding the ID properties of the learningContent resource. If so, please share the CSDL PR with me on the side (Teams). 
3. Related to the changelog entries: please follow the guidelines here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog?anchor=changelist-record-properties

Feel free to ping me on the side if you have any questions. 

Here to help!

Isaac Vargas

File: changelog/Microsoft.Viva.Learning.json [hunk 3]
```diff
@@ -33,6 +33,40 @@
...
+          "Target": "learningContent"
+        },
+        {
+          "Id": "1e87915b-ba79-4d40-9dec-a7777ad64f99",
+          "ApiChange": "PropertyRef",
+          "ChangedApiName": "id",
+          "ChangeType": "Addition",
+          "Description": "Added the [id](https://docs.microsoft.com/en-us/graph/api/resources/learningContent?view=graph-rest-beta) resource",
+          "Target": "learningContent"
+        },
```
Comment (isvargasmsft) [inline changelog/Microsoft.Viva.Learning.json:54]: I am yet to see the CSDL PR with these schema changes (please share them on the side) but, from what I can deduce, these 2 entries are not using the best format. 

Were these changelog entries generated or manually created? 

Guidance to changelog entries here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211742/Changelog?anchor=changelist-record-properties

File: api-reference/beta/api/learningcontent-update.md (hunk 2 above)
Comment (SritejaKampara) [reply api-reference/beta/api/learningcontent-update.md:156]: https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads/pullrequest/6558309

File: changelog/Microsoft.Viva.Learning.json (hunk 3 above)
Comment (angelgolfer-ms) [reply changelog/Microsoft.Viva.Learning.json:54]: @SritejaKampara , I was browsing your PR and saw this reply from you. FYI I will copy your response in an email to let @isvargasmsft know, and then remove the URL you provided that points to an internal location for the CSDL.

File: api-reference/beta/api/learningcontent-get.md (hunk 1 above)
Comment (SritejaKampara) [reply api-reference/beta/api/learningcontent-get.md]: That PR is no more needed. The change is required as externalId is now an alternate key and id would be the primary Key, so externalId='externalId' would be the way of calling it correctly.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [18513](https://github.com/microsoftgraph/microsoft-graph-docs/pull/18513) into this branch to include my final edits.

To avoid confusion, I removed from this PR (17997) changes that will be introduced with PR #18134.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**